src/sparseml/sparsification/modifier_pruning.py

```diff
@@ -26,6 +26,7 @@
     BaseUpdate,
     ModifierProp,
 )
+from sparseml.sparsification.types import SparsificationTypes
```

Comment:
```suggestion
from sparseml.sparsification import SparsificationTypes
```

Reply:
since this is in the same package, we run the risk of a circular dependency being introduce in the future if we inherit from the parent init that we export to